include/irr/asset/IImageAssetHandlerBase.h

```diff
@@ -50,65 +50,93 @@ class IImageAssetHandlerBase : public virtual core::IReferenceCounted
 		/*
 			Patch for not supported by OpenGL R8_SRGB formats.
 			Returns converted image. Input image needs to have
-			all the regions filled and texel buffer as well.
-
-			@devshgraphicsprogramming it should support layers and mipmaps
-			as well for GLI, do we have a filter that can handle it?
+			all the regions filled and texel buffer attached as well.
+			Beware it doesn't validate input image, you have to
+			make sure it's R8 format
```

Comment:
actually I will change it

Reply:
if you use convert filter with templated arguments then it will validate if its R8